jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -125,18 +129,24 @@ def run_tf(mode):
       else:
         assert False
 
-    def is_tf_exception(lim: jax2tf.jax2tf.Limitation):
+    def is_tf_exception(lim: jcs.Limitation):
       return (lim.ErrorType == 'Missing TF support' and
               self.tf_default_device.device_type in lim.Devices)
 
     result_tf = None
     for mode in ("eager", "graph", "compiled"):
-      current_limitations = jax2tf.jax2tf.all_limitations[:]
+      current_limitations = jcs.all_limitations[:]
```

Comment:
nit: perhaps it is worth saving just the `len(current_limitations)`. This makes it a little clearer to the user that the old limitations are not really needed below.

Reply:
Addressed in `d069c18`